src/forms.js

```diff
@@ -32,11 +32,10 @@ fn.extend({
   },
 
   val(value) {
-    if (value === undefined) {
-      return this[0].value;
-    } else {
-      return this.each(v => v.value = value);
-    }
+    return ( value === undefined ?
+      this[0].value :
+      this.each(v => v.value = value)
+    );
   }
```

Comment:
@shshaw ... can you have a look at the `/dist/cash.min.js` file before/after this change... if it is no different, or even worse... I am not in favour of obscuring code without any tangible gains